res/database/Default/database_programmes.xml

```diff
@@ -33508,6 +33539,7 @@ Doborowa obsada.</pl>
 			<title>
 				<de>Longdrink</de>
 				<en>Longdrink</en>
+				<pl>Longdrink - Drink</pl>
```

Comment:
intentional?

Reply:
Most of the videos I try to give the original and Polish name. In this case, longdrink is the drink in Polish.